components/vf-show-more/vf-show-more.njk

```diff
@@ -0,0 +1,48 @@
+{# Make sure any variables are listed inside the following if statement #}
+{% if context %}
+  {# {% set vf-show-more = context.vf-show-more %} #}
+{% endif %}
+
+{# You can do Nunjucks logic in your templates that references the .yml settings
+   What's Nunjucks: https://mozilla.github.io/nunjucks/templating.html #}
+{#
+{% if item.image == "blank" %}
+  <span><!-- no image --></span>
+  {% else %}
+  {{ item.image }}
+{% endif %}
+#}
+
+<section data-vf-js-show-more data-vf-js-show-more-pager-size="3" class="vf-show-more">
+  {% render '@vf-section-header--default', {
```

Comment:
this could be problematic when imported into the 11ty projects. I can change this so it 'follows suit' with other components.

Yeah - I need to document how to create a component now that 11ty has some 'special way'. Because of the `{% render %}`.

Reply:
You've more experience here so I'll defer to you — but in practice I don't think we'll use this NJK template and rather graft the data attributes as needed.

Reply:
If/When it gets documented it should be easy … famous last words.